scitt/create_signed_statement.py

```diff
@@ -18,13 +18,9 @@
 from ecdsa import SigningKey, VerifyingKey
 
 
-# Feed header label comes from version 2 of the scitt architecture document
-# https://www.ietf.org/archive/id/draft-birkholz-scitt-architecture-02.html#name-envelope-and-claim-format
-HEADER_LABEL_FEED = 392
-
 # CWT header label comes from version 4 of the scitt architecture document
-# https://www.ietf.org/archive/id/draft-ietf-scitt-architecture-04.html#name-issuer-identity
-HEADER_LABEL_CWT = 13
+# https://www.ietf.org/archive/id/draft-ietf-scitt-architecture-07.html##name-signed-statements
+HEADER_LABEL_CWT = 15
```

Comment:
Nice! I am glad we are keeping up with the latest spec.

We need to make sure this matches what the backend expects.

The backend currently expects the cwt protected header at key 13.

So we should only merge this once the backend changes to be updated to 15 :)

Reply:
Thanks, @honourfish 
To avoid the "big bang theory", do we want to support both 13 and 15 for a few weeks? This way any github actions that haven't been updated will continue to work.